src/lib/Output/Normalizer/ArrayListNormalizer.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Rest\Output\Normalizer;
+
+use Ibexa\Rest\Output\Generator\Data\ArrayList;
+use Symfony\Component\Serializer\Normalizer\NormalizerAwareInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerAwareTrait;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+final class ArrayListNormalizer implements NormalizerInterface, NormalizerAwareInterface
+{
+    use NormalizerAwareTrait;
+
+    /**
+     * @param \Ibexa\Rest\Output\Generator\Data\ArrayList $object
+     * @param array<mixed> $context
+     *
+     * @return array<mixed>
```

Comment:
```suggestion
     * @return array<mixed>
     *
     * @throws \Symfony\Component\Serializer\Exception\ExceptionInterface
```

Reply:
This is somewhat implied by the interface declaration, right?